ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -35,17 +35,23 @@ cp "${SHARED_DIR}/install-config.yaml" "${dir}/"
 mkdir -p ~/.ssh
 cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
 
-TF_LOG=debug openshift-install --dir="${dir}" create cluster 2>&1 | grep --line-buffered -v password &
+TF_LOG=debug openshift-install --dir="${dir}" create cluster 2>/dev/null
 
 set +e
 wait "$!"
-ret=$?
+ret="$?"
 set -e
 
+if [[ -f "${dir}/.openshift_install.log" ]]; then
```

Comment:
similar to https://github.com/openshift/release/pull/7936/commits/85d0d39cc691ad18b367354e819c0b3c750e5e21#r398234103

Reply:
Dropped the conditional with 1044cf8d1 -> 1a7b314f9.